packages/elements-core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@stoplight/elements-core",
-  "version": "7.1.7",
```

Comment:
Might be worth bumping dev-portal too as it's relevant to ninja, not studio only.

Reply:
God damnit, you are right again 😂